src/tests/cucumber/features/steps/common_steps/steps.py

```diff
@@ -34,6 +36,15 @@ def simu_success(context):
     assert context.return_code == 0
 
 
+@then('the modeler simulation succeeds')
+def simu_success(context):
+    print(f"context.return_code = {context.return_code} ***********************")
+    assert context.return_code == 0
+    context.output_path = os.path.join(context.study_path, "output")  # TODO : fixme parse_output_folder_from_logs(out)
+    context.moh = modeler_output_handler(context.output_path)
```

Comment:
instead of adding this step, just check that return_code==0 before instantiating "moh" in "when i run modeler" step:
- save you from creating a new step
- it is not the role of "then the simulation succeeds" step to import outputs, since the user may chose not to run this check (but they will still need access to the outputs)

Reply:
done